### PR TITLE
The landing page

### PR DESCRIPTION
**Pull request by devmarye**
Here I designed out the landing page, and  it is responsive

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,11 +2,15 @@ import React from 'react';
...
 import './index.css';
 import App from './App';
+import { BrowserRouter, HashRouter } from "react-router-dom";
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+   
+    <HashRouter>
```
Comment (TheoOkafor) [inline src/index.js]: @brendagold Why are you using HashRouter instead of BrowserRouter

File: src/services/api.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const api = axios.create({
+    baseURL:"https://cdn.contentful.com//spaces/nbe8tmkxy59u/environments/master/entries?access_token=ynfiaMt1N5Skp8U-uq7Z49Xc9OSl6g-je6-cEcapCqU&content_type=exhibition"
```
Comment (TheoOkafor) [inline src/services/api.js]: What's this URL for?

**Review by TheoOkafor — COMMENTED** [review]
Looks good. Few comments...

File: src/index.js (hunk 1 above)
Comment (devmarye) [reply src/index.js]: I used Browser router but i am having issues with the routes when I uploaded it to gh pages. So while searching for solutions I saw one that said change browser to hash. But it did not help so i changed it back to browserRouter.

File: src/services/api.js (hunk 2 above)
Comment (devmarye) [reply src/services/api.js]: I copied the code from a previous project I did. I have changed it to the pets API

**Review by TheoOkafor — APPROVED** [review]

**Comment by TheoOkafor** [comment]
Feel free to merge it.

**Pull request merged**